mitodl/mitxpro
# Implemented correct email link for bulk enrollment

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #196

#### What's this PR do?
Updates the bulk enrollment link that we send out and updates the coupon assignment to indicate that it has been redeemed, effectively completing the full enrollment flow for bulk enrollment and providing the necessary data for unused coupon notifications (#277)

#### How should this be manually tested?
- Create a 100% off coupon for some product in the coupon creation form (choose a product that you aren't already enrolled in)
- Go to the bulk enrollment form and send that coupon out, making sure that your email address is the recipient
- Click the link in one of the bulk enrollment emails you receive. This should take you to the checkout page with the correct product selected and a pre-filled 100%-off coupon code
- Complete checkout and check your dashboard. You should see that your user is now enrolled in the given course run/program. You should also see the your `ProductCouponAssignment` has been set to `redeemed=True` (this can be checked in Django admin)

#### Any background context you want to provide?
- For convenience I added a link in the ecommerce admin pages to go back to the ecommerce admin index page (screenshot below)
- I got rid of the option to select "course" as a product type in the front end since that is invalid in this app

#### Screenshots (if appropriate)
![ss 2019-05-28 at 17 01 24 ](https://user-images.githubusercontent.com/14932219/58512105-3fe50180-816a-11e9-8692-852043c57709.png)
